routes/productImage/helpers.js

```diff
@@ -0,0 +1,25 @@
+export default {
+    imageCreate: async (req, res) => {
+        try {
+            //console.log(req.file); // to see what is returned to you
+            const { public_id, url } = await req.file;
+
+            const imageData = {
+                image_id: public_id,
+                image_url: url,
+            };
+            
+            return res.send({ data: imageData, success: true });
+        } catch (err) {
+            res.send({ err, success: false });
+        }
+    },
+
+    // deletes products using id
+    imageDelete: async (req, res) => {
+    },
+
+    // updates products using id
+    imageUpdate: async (req, res) => {
+    },
+};
```

Comment:
`imageDelete` and `imageUpdate` are incomplete, either finish them out or remove them from this PR. You can create a new ticket to address updating and deleting images